libstuff/STCPServer.cpp

```diff
@@ -18,6 +18,8 @@ STCPServer::Port* STCPServer::openPort(const string& host) {
     Port port;
     port.host = host;
     port.s = S_socket(host, true, true, false);
+    port.isHost = true;
```

Comment:
Is this ever false? Seems like it's always true, in which case, we could remove it.